webcomponents/tool/src/main/frontend/js/forms/sakai-text-area.js

```diff
@@ -0,0 +1,53 @@
+import { html } from "../assets/lit-element/lit-element.js";
+import { SakaiElement } from "../sakai-element.js";
+/**
+ * Provides a text area input field, supports the maxlength attribute displaying an errorMessage when the limit is reached by the user.
+ * @example <caption>Usage:</caption>
+ * <sakai-text-area inputId="text-area-story" maxLength="20" cols="2" rows="45" errorMessage="You can't insert more than 20 chars"></sakai-text-input>
+ *
+ */
+
+export class SakaiTextArea extends SakaiElement {
+  static get properties() {
+    return {
+      inputId: {
+        type: String
+      },
```

Comment:
Just collapse these to one line and make sure to used hyphenated attribute names, unless you know that all browsers support camel case. So,
inputId: { attribute: "input-id", type: String },